src/Cake.NuGet/NuGetPackageInstaller.cs

```diff
@@ -175,6 +160,38 @@ public IReadOnlyCollection<IFile> Install(PackageReference package, PackageType
             return result;
         }
 
+        private static DirectoryPath GetPackagePath(DirectoryPath root, PackageReference package)
+        {
+            if (package.Parameters.ContainsKey("version"))
+            {
+                var version = package.Parameters["version"].First();
+                return root.Combine($"{package.Package}.{version}");
```

Comment:
Agreed, nuget.exe doesn't give much info though. hacky way would be ExcludeVersion and have ` root.Combine($"{package.Package}.{version}")` be outputdir 😆 

`nuget install ReportGenerator -Version 2.4.5 -Verbosity detailed` says it installs to 2.4.5 but that's a lie 😉